evap/evaluation/models.py

```diff
@@ -711,9 +717,24 @@ def editor_approve(self):
     def manager_approve(self):
         pass
 
-    @transition(field=state, source=[State.PREPARED, State.EDITOR_APPROVED, State.APPROVED], target=State.NEW)
-    def revert_to_new(self):
-        pass
+    @transition(
+        field=state,
+        source=[
+            State.PREPARED,
+            State.EDITOR_APPROVED,
+            State.APPROVED,
+            State.IN_EVALUATION,
+            State.EVALUATED,
+            State.REVIEWED,
+        ],
+        target=State.NEW,
+    )
+    def reset_to_new(self, delete_previous_answers: bool | None = False):
```

Comment:
Why `| None`?

Reply:
So that one doesn't have to specify `delete_previous_answers=False` for every `.reset_to_new()`. But as every other current usage is in the tests, it will be clearer when this is removed